openprescribing/frontend/views/spending_utils.py

```diff
@@ -116,113 +102,51 @@ def _get_names_for_bnf_codes(bnf_codes):
 
 def _get_concession_cost_matrices(min_date, max_date, org_type, org_id):
     """
-    Given a date range (inclusive) and an organisation return a set of matrices
-    detailing all spending affected by price concessions during the period.
-
-    Returns a ConcessionCostMatrices object with the following attributes:
+    Given a date range (inclusive) and an organisation return three
+    LaballedArrays (of the same shape) detailing all spending affected by price
```

Comment:
`LabelledArray` suggests something 1-dimensional -- why not `LabelledMatrix`?

Reply:
"Matrix" isn't quite right either, as these can be 1-dimensional after summing rows/columns. I was following numpy's "ndarray" terminology, but I'd be open to other suggestions.

Reply:
Summarising our conversation for the record: we've decided that "matrix" is probably the least confusing name, even if it is a bit wrong sometimes